docs/api/index.md

```diff
@@ -960,6 +952,19 @@ When you use `test` in the top level of file, they are collected as part of the
   })
   ```
 
+  You can provide an optional argument to test that a specific error is thrown:
+
+  - regular expression: error message matches the pattern
+  - string: error message includes the substring
+
+  :::tip
+    You must wrap the code in a function, otherwise the error will not be caught, and the assertion will fail.
+  :::
+
+  :::tip
+    To assert that an async function rejects, you must use [`rejects`](#rejects) together with [`toThrowError`](#tothrowerror).  Otherwise Vitest will complain that it caught an unhandled error during the test run.
```

Comment:
```suggestion
    To assert that an async function rejects, you must use [`rejects`](#rejects) together with [`toThrowError`](#tothrowerror).  Otherwise, Vitest will complain that it caught an unhandled error during the test run.
```